templates/commands/goldentest/test_funcs.go

```diff
@@ -79,12 +79,13 @@ func parseTestCases(ctx context.Context, location string, testNames []string) ([
 
 	if len(testNames) > 0 {
 		for _, testName := range testNames {
-			testCase, err := buildTestCase(ctx, testDir, testName)
-			if err != nil {
-				return nil, err
+			if _, err := os.Stat(filepath.Join(testDir, testName)); err == nil {
```

Comment:
This code ignores the error returned from os.Stat(). We should be checking whether the error is os.ErrNotExist (which is fine) or some other random error (which isn't fine and should be returned).